dave_demos/README.md

```diff
@@ -0,0 +1,35 @@
+## Examples
+
+### 1. Launching a Dave Model using Fuel URI
+
+To launch a Dave model directly from a Fuel URI, follow these steps:
+
+1. Build and source the workspace:
+   ```bash
+   colcon build && source install/setup.bash
+   ```
+2. Launch the model using the specified launch file:
+   ```bash
+   ros2 launch dave_demos mossy_cinder_block.launch.yaml
+   ```
+
+This method simplifies the process by pulling the model directly from Fuel, ensuring you always have the latest version without needing to manage local files.
+
+### 2. Launching a Dave Model using Downloaded Model Files
+
+If you prefer to use model files downloaded from Fuel, proceed as follows:
+
+1. Set the resource path to your local model files:
+   ```bash
+   export GZ_SIM_RESOURCE_PATH=<Path-to-dave_ws>/src/dave/dave_model_description
```

Comment:
https://github.com/gazebosim/ros_gz/pull/492 Could you try this to get rid of this step?

Reply:
I've no experience with `hooks` but it seems it's another alternative
Reference: https://github.com/gazebosim/ros_gz/tree/ros2/ros_gz_sim_demos

Also... hard to ask, but with best wishes, it seems `ros_gz`'s demo is now all in python. What do you think of working with python from this and now on?

Reply:
All other parts worked fine! Thank you so much for investigating the previous dave structure and attempt to follow how it was done 👍

Reply:
Sure thing, I'll make those changes!